packages/components/src/dateInput/DateInput.js

```diff
@@ -1,90 +1,124 @@
-    this.props.onDateChange(date);
-    this.hideDatePicker();
+  handleYearChange = (year: string) => {
+    this.props.onDateChange(year, 'year');
   };
 
   render() {
-    const { isPickerVisible } = this.state;
-    const {
-      label,
-      formLabelContainerStyle,
-      formLabelTextStyle,
-      date,
-      defaultDate,
-      dateFormat,
-      placeholder,
-      iconName,
-      confirmLabel,
-      cancelLabel,
-    } = this.props;
-
     return (
-      <View>
-        {label != null && (
-          <FormLabel
-            containerStyle={formLabelContainerStyle}
-            labelStyle={formLabelTextStyle}
-          >
-            {label}
-          </FormLabel>
-        )}
-        <PickerButton
-          value={date ? format(date, dateFormat) : null}
-          placeholder={placeholder}
-          onPress={this.showDatePicker}
-          iconName={iconName}
-        />
-        <DatePicker
-          isVisible={isPickerVisible}
-          labels={{
-            cancel: cancelLabel,
-            confirm: confirmLabel,
-          }}
-          date={date ?? defaultDate}
-          onConfirm={this.handleDateChange}
-          onDismiss={this.hideDatePicker}
-        />
+      <View style={styles.container}>
+        <Text style={styles.text}>Date of birth</Text>
+        <View style={styles.dateFormWrapper}>
+          <View style={styles.dayPickerWrapper}>
+            <TextInput
+              onChangeText={this.handleDayChange}
+              placeholder="DD"
+              autoCorrect={false}
+              value={this.props.date.day}
+              error={this.props.errors.day}
+              confirmLabel="OK"
+              type="number"
+            />
+          </View>
+          <View style={styles.monthPickerWrapper}>
+            <Picker
+              selectedValue={this.props.date.month}
+              optionsData={this.getMonthOptions()}
+              onValueChange={this.handleMonthChange}
+              placeholder="Month"
+              confirmLabel="OK"
+            />
+          </View>
+          <View style={styles.yearPickerWrapper}>
+            <TextInput
+              onChangeText={this.handleYearChange}
+              autoCorrect={false}
+              value={this.props.date.year}
+              error={this.props.errors.year}
+              placeholder="YYYY"
+              confirmLabel="OK"
+              type="number"
+            />
+          </View>
+        </View>
       </View>
     );
   }
 }
+
+export default DateInput;
+
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    flexDirection: 'column',
+    marginTop: parseInt(defaultTokens.spaceSmall, 10),
+  },
+  dateFormWrapper: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    flexDirection: 'row',
+  },
+  text: {
+    marginStart: parseInt(defaultTokens.spaceSmall, 10),
+    marginBottom: parseInt(defaultTokens.spaceXXSmall, 10),
+  },
+  dayPickerWrapper: {
+    flex: 0.2,
```

Comment:
What those `0.x` values mean?

Reply:
This is for proportionally dividing columns

Reply:
Why not to use it without `0.`. It should work with the same ratio principle
Like 
```
dayPickerWrapper:
  flex: 2

monthPickerWrapper: 
  flex: 4

yearPickerWrapper: 
  flex: 3
```
means ratio `2:4:3` which is the same like `0.2:0.4:0.3`

Reply:
<img width="413" alt="Screenshot 2019-06-06 at 12 54 32" src="https://user-images.githubusercontent.com/25099820/59027830-4eae7100-885a-11e9-83af-2ab6fb7eecf6.png">
It starts to look like this without 0's